WBooks/Library/MainMenuViewModel.swift

```diff
@@ -12,20 +12,24 @@ class MainMenuViewModel {
     
     var books: [BookViewModel] = []
     
+    var menuController: MainMenuController?
```

Comment:
View Models shouldn't know anything about viewControllers. Only the View Controller should has a reference of its ViewModel. 
Here you are creating a "loop" with the references, because the viewController has a reference of the viewModel and vice versa. This creates a memory leak.